.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and test
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
There's no `main` on this repo, and we want to run it on all branches, consolidate.

```
on: [push, pull_request]
```

Reply:
That is what i had initially but with that setting the workflow runs 2x on every push to an open pull request. I can remove `main` for sure.

Reply:
I would not worry about it, everybody works off forks and these are kinda "separate" runs.